### ISSUE TITLE
Assets are not freely licensed

### ISSUE THREAD

**Issue by alexgleason** [TEAMarg/ID-46-Arduventure#1]
Currently there is this line in the README:

> Story, characters, sprites, tiles, design and art: copyrighted to TEAM a.r.g. NOT under MIT !

I am hoping you'll reconsider this, as high-quality Open Source games are severely underrepresented and this is a great opportunity to release a fully Open Source game.

I'm actually not sure what the motivation is for keeping the assets proprietary. Why go through the trouble of creating an Open Source game on an open hardware platform only to leave the job unfinished?

The game's graphics are very similar to Nintendo games like Pokemon Yellow. It has beautiful graphics, but not a breakthrough in video game art. I don't understand what the benefit of keeping this proprietary is, this is not a new or revolutionary idea that everyone will want to steal.

![town](https://user-images.githubusercontent.com/3639540/39504589-946839de-4d9a-11e8-8d32-6b857cef120c.png)

Also, I expect Nintendo's games to be proprietary. They don't share, and it's why I stopped playing their games. I do expect members of the Open Source community to share. Meanwhile the developers of [Children of Solarus](http://www.solarus-games.org/blog/) are breaking their backs creating freely-licensed graphics to replace the Zelda sprites that Nintendo refuses to share with the community. Releasing a MIT-licensed game with proprietary assets seems like a major step back to me.

![1 -18-01-21](https://user-images.githubusercontent.com/3639540/39504604-ab95d27e-4d9a-11e8-88d2-b15071a74b92.png)

When people don't freely-license assets, all they are doing is creating extra work for the community who will have to fork the game and create new assets if they want the freedom to run the game how they wish.

If you simply don't want others to use the name "Arduventure" without your permission, that is understandable (because of trademarks) and it's common for Open Source projects to restrict how people can use their name and logo (FireFox has this, for instance).

Thanks!

**Comment by jorng** [comment]
Seems pretty straight forward. If you want to fork and make your own game, create your own artwork.

**Comment by JO3RI** [comment]
The reason why? Because we had one of our games taken by a game company, that just took the game, replaced it's name and have put their own company log and name on it with **no reference** in the game, on their website nor on the box and distributed it as their own game for promoting their company.

**Comment by yoooooooooooooooooooooooo** [comment]
@JO3RI that's why copyleft licenses exist. If you licensed the artwork under CC-BY-SA or GPL or something else, other people can reuse the artwork but have to credit you. Under a copyleft license, "took the game, replaced it's own name and have put their own company logo on it with no reference in the game, on their website nor on the box and distributed it as their own game for promoting their company" would be flat out illegal.

**Comment by disk0x** [comment]
> The reason why? Because we had one of our games taken by a game company, that just took the game, replaced it's own name and have put their own company logo on it with no reference in the game, on their website nor on the box and distributed it as their own game for promoting their company.

But this is a license violation, you had a way to force them respect your license! Even permissive MIT that you use for the source code has this (let alone copyleft licenses which offer you even more protection):

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

You chose not to enforce your copyright in the case you described, and the same could happen to you with All Rights Reserved if you don't enforce your copyright.

**Comment by Gaveno** [comment]
I guess for me I'm just not really understanding what people are losing out on here. People are able to mod the game for themselves and their education. If they can change the graphics and some text and create their own RPG they can. At some point we may release the World Editor tool to allow open world games to be easily made with your own graphics. Using the same graphical assets for multiple games reduces the originality for all of them. If anyone wants to release a mod of the game for non-commercial use and they contact us I'm sure we can figure it out.

**Comment by alexgleason** [comment]
Hi Gavin!

> Using the same graphical assets for multiple games reduces the originality for all of them.

I'm not sure I agree with this. While there's some inherent truth to it, there are many examples where original experiences were made by reusing assets. For example, Link's Awakening had a lot of its graphics reused in Oracle of Seasons/Ages, but these are *very* different games.

![zelda-links-awakening](https://user-images.githubusercontent.com/3639540/39537739-7629133a-4e08-11e8-9a35-5e68cb83fc45.gif)
![proxy duckduckgo com](https://user-images.githubusercontent.com/3639540/39537742-79ee9fbc-4e08-11e8-8c5a-6d7daa92b861.jpeg)

Other examples:

* Zelda Majora's Mask was made up almost entirely of Ocarina of Time assets, and again the games are totally different experiences.
* Super Smash Bros was created by combining art from many different games to create an entirely new game which is nothing like the individual games.

There's fan-game Pokemon Insurgence which Kotaku described as [More Exciting Than The Real Thing](https://kotaku.com/fan-made-pokemon-game-is-more-exciting-than-the-real-th-1655072789). Then of course there was the highly anticipated [Pokemon Uranium](http://pokemon-uranium.wikia.com/wiki/Main_Page) fan-game which, after being developed for **nine years** was shut down by Nintendo on day 1 of its release.

So I think it is totally possible to reuse assets and create a unique experience. That said, if someone reuses the assets and creates a bland, uninteresting experience, so what? [Copying creates something new, it takes nothing away](https://www.youtube.com/watch?v=IeTybKL1pM4). Of course you should always [give credit where credit is due](https://www.youtube.com/watch?v=dPtH2KPuQbs), which I think is the problem TEAM a.r.g. ran into.

---

> I'm just not really understanding what people are losing out on here.

My concern is that, aside from releasing the code for the game, this game is not fundamentally different than a Nintendo game in terms of the freedoms it affords its users. Open Source games are licensed in such a way that they are partially owned by the public at large, and (aside from its code) this game is not.

This is my suggestion, which I think would solve both our problems:

1. The code for the game is licensed under MIT or GPL. I'd choose GPL to force others to share back.
2. The assets are licensed under [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) which requires credit to be given to you and modifications to be shared back with the community.
3. The name Arduventure and the Arduventure logo are trademarks of TEAM a.r.g. and cannot be used without permission.

**Comment by alexgleason** [comment]
@jorng, I am maintaining [a number of Open Source packages](https://github.com/alexgleason?utf8=%E2%9C%93&tab=repositories&q=&type=source&language=) on GitHub. In addition, I co-founded [a web development agency](https://candlewaster.co/) that creates exclusively Open Source works and I've created [an illustrated storybook](http://toddlertimothy.com/) which I licensed under CC-BY-SA. Finally, I am a contributor to the [Arcmage](http://arcmage.org/) project which is a GPL card game with custom art. I have revolved my entire life around creating only freely-licensed creations. I do not have time to create my own game including assets, which is why I rely on other projects to reciprocate in this [gift economy](https://en.wikipedia.org/wiki/Gift_economy). I am certainly doing the best that I can to give back for all the people who have given to me.

Next, I have never, nor do I have the authority to, force my preferences upon anyone else. I've kept a non-aggressive but honest tone throughout this discussion. I do not expect the developers to freely license this work, nor do I feel entitled to it. I am raising this issue because it's something I care about and hope that others may hear me out on and form the same conclusions that I have.

**Comment by alexgleason** [comment]
I just want to say that I bought 2 Arduboys specifically because of this game. I was checking the forum every single day for the release of this game. It was finally released in May, and I was extremely excited. When I found out that the assets were proprietary, I chose not to play it. It's been two months now, and my girlfriend and my Arduboys are sitting in the drawer. I'm not going to play this game, and I feel turned off on the Arduboy in general.

I don't think you owe me anything, but I want you to know this.
